code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -130,3 +130,8 @@
 		for(var/obj/item/weapon/bomberman/dispenser in src)
 			if(dispenser.spam_bomb)
 				dispenser.attack_self(src)
+
+		if(get_active_hand() && istype(get_active_hand(),/obj/item/weapon/rcl))

```

Comment:
You call get_active_hand 3 times.

Put var/obj/item/weapon/rcl/R = get_active_hand() before the if statements, then use if(R && istype(R) && R.active)